src/logic/wallets/getWeb3.js

```diff
@@ -170,8 +170,12 @@ export const getProviderInfo: Function = async (
 
 export const getAddressFromENS = async (name: string) => {
   const ens = new ENS(web3)
-  const address = await ens.resolver(name).addr()
-
+  let address = null
+  try {
+    address = await ens.resolver(name).addr()
+  } catch (error) {
+    console.error('Error fetching resolved DNS', error)
```

Comment:
How do you detect if there was an error or the address is just not set?

Reply:
IIRC it would throw even if the address is not set for a name. I wouldn't say it's an error. I would leave a way for whatever uses this function to catch the error itself and display appropiate error message. This is why I didn't use try/catch here

Reply:
done